test/workers/repository/updates/branchify.spec.ts

```diff
@@ -1,32 +1,28 @@
-/** @type any */
-let config;
+import { RenovateConfig, defaultConfig, mocked } from '../../../util';
+import { branchifyUpgrades } from '../../../../lib/workers/repository/updates/branchify';
+import * as _flatten from '../../../../lib/workers/repository/updates/flatten';
+import { BranchConfig } from '../../../../lib/workers/common';
+
+const flatten = mocked(_flatten);
```

Comment:
```suggestion
const flattenUpdates = mocked(_flatten).flattenUpdates;
```

Reply:
`TypeError: flattenUpdates.mockReturnValueOnce is not a function`

Reply:
You need the jest.mock change below too